calicoctl/calico_ctl/checksystem.py

```diff
@@ -224,8 +227,11 @@ def check_etcd_version():
         detected_version_string_raw = \
             client.etcd_client.api_execute("/version", 'GET').data
     except EtcdConnectionFailed:
+        etcd_env = os.getenv(ETCD_ENDPOINTS_ENV, "")
```

Comment:
I don't care, so I've deleted it; but in pretty much all the other places we call this function we pass "" in.